ui/scss/component/_content.scss

```
@@ -260,7 +260,7 @@
     }
   }
   .content__info {
-    &:not(.content-info__playlist) {
+    &:not(.content-info__playlist):not(.content__info--shorts-floating) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)